.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: Task - Build Madara
+on:
+  workflow_dispatch:
+  workflow_call:
+jobs:
+  build:
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+      - uses: Swatinem/rust-cache@v2
+      - uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: 1.78.0
+      - name: Setup build deps
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y clang llvm libudev-dev protobuf-compiler
+      - uses: rui314/setup-mold@v1
+      - uses: software-mansion/setup-scarb@v1
```

Comment:
Any reason why you are calling 2 times software-mansion/setup-scarb@v1 for version 2.8.2?

Reply:
my bad, removed